### PR TITLE
Update Appian

### PR DESCRIPTION
**Pull request by t-miller**
1. Updated logo to new version
2. Updated regex to remove `/suite/`, matching pattern shown in [Appian's docs](https://docs.appian.com/suite/help/22.3/Embedded_Interfaces.html#load-appian's-web-components)

### CODE REVIEW

File: src/technologies/a.json [hunk 1]
```diff
@@ -3184,7 +3184,7 @@
     "description": "Appian is an enterprise low-code application platform.",
     "icon": "Appian.png",
```
Comment (nurbek91) [inline src/technologies/a.json:3185]: preferred format ымп
[Appian.svg.zip](https://github.com/wappalyzer/wappalyzer/files/9464940/Appian.svg.zip)

File: src/technologies/a.json [hunk 2]
```diff
@@ -3184,7 +3184,7 @@
...
     "saas": true,
-    "scriptSrc": "/suite/tempo/ui/sail-client/embeddedBootstrap.nocache.js",
+    "scriptSrc": "tempo/ui/sail-client/embeddedBootstrap.nocache.js",
```
Comment (nurbek91) [inline src/technologies/a.json:3187]: `"scriptSrc": "tempo/ui/sail-client/embeddedBootstrap\\.nocache\\.js",`

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks for your PR!!!
I could not find examples, there are no results in the Wappalyzer database
Do you have any links I would like to look at possible additional traces
https://www.wappalyzer.com/technologies/paas/appian/

**Comment by t-miller** [comment]
> Hey mate, thanks for your PR!!! I could not find examples, there are no results in the Wappalyzer database Do you have any links I would like to look at possible additional traces https://www.wappalyzer.com/technologies/paas/appian/

Hey @nurbek91,

I stood up a Community site and provisioned a user with basic permissions that should allow you to view the detection - I'll inactivate the user later.

site: `https://xloi.appian.community/suite/sites/net-zero-carbon/page/overview`
username: `test.user`
pasword: `test123`

**Comment by nurbek91** [comment]
Hey @t-millerm thanks!!! We can add this in addition, maybe it will help 
```
"js": {
      "APPIAN": "",
      "Appian": "",
      "webpackJsonpAppian": "",
      "_APPIAN_PROXIES_INITIALIZED": ""
    },
```

**Review by nurbek91 — APPROVED** [review]
thanks mate!!

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**